internal/provider/waypoint/resource_waypoint_template.go

```diff
@@ -118,7 +126,7 @@ func (r *TemplateResource) Schema(ctx context.Context, req resource.SchemaReques
 			},
 			"readme_markdown_template": schema.StringAttribute{
 				Optional:    true,
-				Description: "Instructions for using the template (markdown format supported",
+				Description: "Instructions for using the template (markdown format supported). Must be base 64 encoded.",
```

Comment:
I don't think this is true anymore?

Reply:
Ah yes good catches, I fixed those in the old commit and missed them when going back through and reverting the project_id changes today, I'll clean those up while I'm shortening the commit history.